.travis.yml

```diff
@@ -6,6 +6,9 @@ services:
 go:
   - "1.9"
   - "1.10"
```

Comment:
can we drop 1.9 and 1.10?

Reply:
I don't think we can delete them now, because it can bring some errors if anybody will use functions from new versions of Golang in atlas-app-toolkit. Now we don't have some specific requirements for Golang version due to this we keep old versions for  a while, and gonna think how to deprecate supporting of old Golang version In future